Gemfile

```diff
@@ -7,6 +7,6 @@ group :local_development do
   gem 'guard-rspec'
   gem 'appraisal'
   gem 'rake'
-  gem 'sqlite3'
+  gem 'sqlite3', '~> 1.4'
```

Comment:
Some tests were failing for different reasons and this is required to make them pass